src/panels/config/person/dialog-person-detail.ts

```diff
@@ -228,9 +325,70 @@ class DialogPersonDetail extends LitElement {
     this._picture = (ev.target as HaPictureUpload).value;
   }
 
+  private async _changePassword() {
+    if (!this._user) {
+      return;
+    }
+    const credential = this._user.credentials.find(
+      (cred) => cred.type === "homeassistant"
+    );
+    if (!credential) {
+      showAlertDialog(this, {
+        title: "No Home Assistant credentials found.",
```

Comment:
I think this is the only one missing localize now

Reply:
Should not happen anyway 😄